public/data/HAT52.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Hamza Ali",
+    "type": "personal",
+    "bio": "Tech Enthusiast!",
+    "avatar": "https://github.com/HAT52.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/HAT52",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Hami_Ali98",
+        "icon": "twitter"
+      },
+      {
+        "name": "Let's connect on LinkedIn",
+        "url": "linkedin.com/in/hamza-ali-8b79171aa",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Let's connect on LinkedIn",
+        "url": "linkedin.com/in/hamza-ali-8b79171aa",
+        "icon": "linkedin"
```

Comment:
You now have your linkedin profile twice, do you want to delete one of these?

Reply:
YES